app/models/tool.rb

```diff
@@ -54,8 +55,8 @@ def check_health
       when :github
         contents = JSON.parse RestClient.get "https://api.github.com/repos/#{repo_name}/contents",
         {:params =>
-          {:client_id => ENV['ST_GITHUB_CLIENT_ID'],
-          'client_secret' => ENV['ST_GITHUB_CLIENT_SECRET']
+          {:client_id => ENV['GITHUB_CLIENT_ID'],
+          'client_secret' => ENV['GITHUB_CLIENT_SECRET']
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.